src/App.js

```diff
@@ -47,14 +47,6 @@ export const App = () => {
 		scriptHash: Neon.create.account(process.env.REACT_APP_NEOFS_ACCOUNT).scriptHash,
 	});
 
-	const [ContentTypeHeader] = useState("Content-Type");
-	const [AuthorizationHeader] = useState("Authorization");
-	const [BearerOwnerIdHeader] = useState("X-Bearer-Owner-Id");
-	const [BearerForAllUsers] = useState("X-Bearer-For-All-Users");
-	const [BearerSignatureHeader] = useState("X-Bearer-Signature");
-	const [BearerSignatureKeyHeader] = useState("X-Bearer-Signature-Key");
-	const [BearerLifetime] = useState("X-Bearer-Lifetime");
```

Comment:
Not sure why you're removing these constants.

Reply:
I don't think it's best practice to do that, I've never seen it. Also, now we have less queries and there is no need to store these constants